solutions/fully-configurable/variables.tf

```diff
@@ -58,18 +58,7 @@ variable "access_key" {
 
 variable "access_key_secret_name" {
   type        = string
-  description = <<-EOT
-    The name of a Kubernetes or OpenShift Secret that contains the Sysdig agent access key under the key `access-key`.
-
-    This variable allows you to reference an existing secret in your cluster, rather than providing the access key directly in your Terraform configuration. This is recommended for improved security and to avoid exposing sensitive credentials in your Terraform state files.
-
-    - If set to a non-null value, the agent will use the access key from the specified secret.
-    - If set to `null` (the default), the value from the `access_key` variable will be used instead.
-
-    Example usage:
-      - Set to `"sysdig-agent-access"` if you have a secret named `sysdig-agent-access` in your target namespace.
-      - The secret must contain a data entry named `access-key`.
-  EOT
+  description = "The name of a Kubernetes or OpenShift Secret that contains the Sysdig agent access key under the key `access-key`. This variable allows you to reference an existing secret in your cluster, rather than providing the access key directly in your Terraform configuration. This is recommended for improved security and to avoid exposing sensitive credentials in your Terraform state files."
```

Comment:
I'm not following why you say "avoid exposing sensitive credentials in your Terraform state files"? The value of the secret will be in the state file right since its passed using the `access_key` input ??